recipe/migrations/osx_arm64.txt

```diff
@@ -73,9 +73,12 @@ wrapt
 mne-base
 mne
 mne-qt-browser
+mne-installer-menus
```

Comment:
I don't understand...this one is noarch?

Reply:
I'm sorry, you are right – I will remove this one from the PR!

Reply:
To be clear, based on that and its dependencies, I'd be shocked to find out this is not installable on osx arm64 right now.

Reply:
I just checked, and indeed it can be installed on `osx-arm64` :)